### DISCUSSION TITLE
Project setup, build and debug related.

### DISCUSSION THREAD

**Discussion by shubhamshirude2512** [ksh93/ksh#643] [Q&A]
Hello, I just cloned this repository on linux and want to know how do I build the project and debug?
Also which IDE will be the best suited for this repository?

**Comment by McDutchie** [comment]
To build:
```sh
bin/package make
```
To debug:
```sh
bin/package use
```

README.md and src/cmd/ksh93/README have more detailed building instructions. If there is anything in there that is not clear to you, please let me know.

I do not use an IDE. Best suited is your favourite editor (vi, emacs, joe, etc.) and the shell.
